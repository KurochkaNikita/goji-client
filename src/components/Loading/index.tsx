import CircularProgress from "@mui/material/CircularProgress";
import Grid from "@mui/material/Grid2";

function Loading() {
  return (
    <Grid
      container
      spacing={0}
      direction="column"
      alignItems="center"
      justifyContent="center"
      sx={{ minHeight: "50vh" }}
    >
      <CircularProgress size="3rem" />
    </Grid>
  );
}

export default Loading;
