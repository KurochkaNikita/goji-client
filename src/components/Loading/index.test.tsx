import { render } from "@testing-library/react";

import Loading from "./index";

describe("Loading", () => {
  it("Snapshot", () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
