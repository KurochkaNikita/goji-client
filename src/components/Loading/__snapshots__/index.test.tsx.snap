// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Snapshot 1`] = `
<div>
  <div
    class="MuiGrid2-root MuiGrid2-container MuiGrid2-direction-xs-column css-6garps-MuiGrid2-root"
  >
    <span
      class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-1wxecgq-MuiCircularProgress-root"
      role="progressbar"
      style="width: 3rem; height: 3rem;"
    >
      <svg
        class="MuiCircularProgress-svg css-54pwck-MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-960umw-MuiCircularProgress-circle"
          cx="44"
          cy="44"
          fill="none"
          r="20.2"
          stroke-width="3.6"
        />
      </svg>
    </span>
  </div>
</div>
`;
